config/services.yaml

```diff
@@ -65,5 +65,10 @@ services:
         arguments:
             $projectDir: '%kernel.project_dir%'
 
+    App\EventSubscriber\RateLimitSubscriber:
+        arguments:
+            $rateLimit: '%env(int:APP_RATE_LIMIT)%'
+            $rateDuration: '%env(int:APP_RATE_DURATION)%'
+
```

Comment:
This section needs to be specified in `config/services_test.yaml` as well. I'm pretty sure there's a way to share these, but it's not set up at the moment. :(

This is where the CI failure is coming from.

Reply:
Ah, that explains it. Done!